plugins/PY/pinpointPy/libs/_pyRedis/PyRedisPlugins.py

```diff
@@ -62,8 +68,9 @@ def onBefore(self, parentId, *args, **kwargs):
             pipeLine = args[0]
             # fixed: Redis Collections Not release #612
             # @quicksandznzn
+            connection_kwargs = pipeLine.connection_pool.connection_kwargs
             pinpoint.add_trace_header(
```

Comment:
must default value 
like this
` pinpoint.add_trace_header(
                Defines.PP_DESTINATION, format_host(connection_kwargs.get('host','localhost'),
                                                    connection_kwargs.get('port',6379),
                                                    connection_kwargs.get('db',0))
                , trace_id)`

Reply:
@eeliu

Reply:
I have a questions,  this dst value look at only show in apm web ,the java code call redis show this , python code why need host port db
<img width="241" alt="image" src="https://github.com/pinpoint-apm/pinpoint-c-agent/assets/23514918/e688116b-e521-4c93-984c-031c457e9ca7">

Reply:
If you don't mind. please add my wechat account **Quicksand_ZN**

Reply:
> connection_kwargs

I will add them.